README.md

```diff
@@ -116,7 +108,7 @@ cd "${CQ_WORKSHOP_DIRECTORY}/camel-quarkus-workshop"
 mvn quarkus:go-offline -fae
 mvn dependency:go-offline -fae
 mvn clean package -fae
-cd part-2-jvm-mode
+cd part-6-jvm-mode
```

Comment:
Here, if we do `cd part-5-kafka/` instead. Then the right docker images should be pulled for free ?

Reply:
yes